oelint_adv/__main__.py

```diff
@@ -142,7 +142,7 @@ def main():
                         fixedfiles += r.fix(f, stash)
                     issues += r.check(f, stash)
                 if not args.quiet:
-                    print("{}/{} files checked".format(index, len(_files)))
+                    print("{}/{} files checked".format(index+1, len(_files)))
```

Comment:
Agree - since the grouping of input files was added the info is not correct anyway - therefore I would rather remove the whole print statement, or is this info somewhat useful to you?

Reply:
Well, in general I find feedback on progress valuable. But if it's not correct anymore or obsolete better to remove it. Changed that line.

Reply:
Thanks - will merge it now